### ISSUE TITLE
Would backport of chain_width to 1.4.x be considered?

### ISSUE THREAD

**Issue by ctz** [rust-lang/rustfmt#4769]
Hello,

Thank you for your work on rustfmt!

Over in rustls we're using rustfmt 2.0 (built from source), because we're using the `chain_width` option reintroduced in this version. I've read in #4582 the issues with releasing/stabilising 2.0, and these seem clear to me.

However, would the maintainers accept a PR that backported support for `chain_width` to the 1.4 release line?

Thanks,
Joe

**Comment by calebcartwright** [comment]
Absolutely! I was actually already planning to backport the changes that made the granular `*_width` options (includes `chain_width`) public and available to users.

Doing so is second on my rustfmt code/development to-do list, though most of my bandwidth these days is preoccupied with the typical admin/support/maintenance/etc. work so not sure exactly when I'd be able to get around to it myself.

PRs are always appreciated as well, but this is definitely something that will get done either way.
